donate/templates/fragments/donate_form_fundraise_up.html

```diff
@@ -0,0 +1,3 @@
+<div class="donate-form__fundraiseup-form">
```

Comment:
Since we are using the same FRU form across both pages, I made this template to reduce the chance of one page missing an update, in case we ever need to update the forms ID.

Reply:
This is clever!